content/source/docs/cloud/integrations/service-now/service-catalog.html.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: "cloud"
+page_title: "Service Catalog - ServiceNow Service Catalog Integration - Terraform Cloud and Terraform Enterprise"
+description: |-
+  Configure Service Catalogs and VCS repositories to allow end users to
+  provision infrastructure using ServiceNow and Terraform Cloud.
+---
+
+# Service Catalog Configuration
+
+When using ServiceNow with the Terraform Cloud integration, you will configure
+at least one Service Catalog. You will also configure one or more Version
+Control System (VCS) repositories containing the Terraform configuration which
+will be used to provision that infrastructure. End users will request
+infrastructure from the Service Catalog, and Terraform Cloud will fulfill the
+request by creating a new Workspace, applying the configuration, and then
+reporting the results back to ServiceNow.
+
+## Prerequisites
```

Comment:
The list items below here are inconsistent. First one is a sentence describing a requirement, next two are just names of objects.

Reply:
Hm. Good point. I went with the following, but it still seems a little awkward. Open to suggestions.

```
1. The Terraform Cloud integration software installed and configured on
   your ServiceNow instance. These steps are covered in the [installation
   documentation](./index.html).
```